content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,132 @@
+# Charte Code de Conduite Contributeurs
+
+## Notre engagement
+
+En tant que membres, contributeurs et dirigeants, nous nous 
+engageons à rendre la participation dans notre communauté 
+une expérience sans harcelement pour tous, quel que soit l'âge, 
+la physionomie, la présence d'un handicap visible ou invisible, l'ethnie, 
+les caractéristiques sexuelles, l'identité et l'expression de genre, 
+le niveau d'expérience, l'éducation, le statut socio-économique, 
+la nationalité, l'apparence personnelle, la race, la religion, 
+ou l'identité et l'orientation sexuelle.

```

Comment:
```suggestion
ou l'identité et l'orientation sexuelle des participant·e·s.
```

Reply:
I think we can go even further and use the inclusive writting all over the Code of Conduct!
I was just thinking of proposing it.

Reply:
Massively in favor of using inclusive language. Don't think adding "des participant·e·s" here is worth it, though.

Reply:
Imho, it could be great. 
However, I'm pretty new to the use of the inclusive language.
Why do you think it's not worth it in this case?

Reply:
To clarify: yes to inclusive language across the whole doc. However adding “les participants” to this sentence isn’t necessary.

Reply:
Okay, my bad, I thought they were already here !
Indeed, I'm not sure if adding "les participants" is worth it.